asabiris/formatter/jinja/service.py

```diff
@@ -28,6 +29,8 @@ def __init__(self, app, service_name="JinjaService"):
 			self.Variables = {}
 
 		self.Environment = jinja2.Environment()
+		# Inject 'now' function into the Jinja2 template's global namespace
+		self.Environment.globals['now'] = lambda: datetime.datetime.utcnow()
```

Comment:
`now` contains the start time of the service?

That's not correct.

Reply:
@ateska From the timestamp they have different time generated at different times

<img width="997" alt="Screenshot 2024-06-07 at 2 19 56 PM" src="https://github.com/TeskaLabs/asab-iris/assets/17985398/e9070c8a-4bbe-4a49-996d-46e5d3a4dff5">